common/components/deploy_model/README.md

```diff
@@ -0,0 +1,152 @@
+# Model Deployment Component
+This component can be used in [azure machine learning pipelines](https://learn.microsoft.com/en-us/azure/machine-learning/concept-ml-pipelines?view=azureml-api-2) to [deploy](https://learn.microsoft.com/en-us/azure/machine-learning/how-to-deploy-online-endpoints?view=azureml-api-2&tabs=azure-cli) the model in [workspace](https://learn.microsoft.com/en-us/azure/machine-learning/concept-workspace?view=azureml-api-2).
```

Comment:
Please change every instance of "azure machine learning" in this PR to "Azure Machine Learning".

Reply:
Done. Thank you @elesel